test/e2e/testing-manifests/storage-csi/hostpath/hostpath/csi-hostpathplugin.yaml

```diff
@@ -15,7 +15,7 @@ spec:
       hostNetwork: true
       containers:
         - name: driver-registrar
-          image: gcr.io/gke-release/csi-driver-registrar:v1.0.0-gke.0
+          image: gcr.io/gke-release/csi-driver-registrar:v1.0.1-gke.0
```

Comment:
what changed in this image?

Reply:
Driver-registrar was erroneously still registering supporting "0.2" and "0.3" csi spec versions.
https://github.com/kubernetes-csi/driver-registrar/pull/74